chromadb/api/client.py

```diff
@@ -79,7 +79,7 @@ def _get_identifier_from_settings(settings: Settings) -> str:
                 identifier = (
                     "ephemeral"  # TODO: support pathing and  multiple ephemeral clients
                 )
-        elif api_impl == "chromadb.api.fastapi.FastAPI":
+        elif "FastAPI" in api_impl:
```

Comment:
nit - can we just ad a special case for Async to make this more explicit? Don't like the subtle magic here

Reply:
I don't either, but it's either this or explicitly handling the case of `chromadb.test.conftest.AsyncFastAPISync`, which would mean shipping a reference to `chromadb.test.conftest` (see https://github.com/chroma-core/chroma/blob/d3a059be5be4ae029bc001eac3e2694653e35e95/chromadb/test/conftest.py#L350). If you think that's preferable it's fine with me.

Reply:
cea9ccfb